components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/policy.js

```diff
@@ -136,12 +136,12 @@ policyModule = function () {
         var mqttBrokerPort = mqttEndPointDeviceConfig.getPort();
         var mqttQueueEndpoint = mqttBrokerURL + ":" + mqttBrokerPort;
 
-        var mqttsenderClass = Packages.org.wso2.carbon.device.mgt.iot.mqtt.PolicyPush;
-        var mqttsender = new mqttsenderClass();
+        var mqttSenderClass = Packages.org.wso2.carbon.device.mgt.iot.mqtt.PolicyPush;
+        var mqttSender = new mqttSenderClass();
```

Comment:
It's not recommended to access any component level classes directly from jaggery runtime. 
What if the jaggery app is hosted as a separate webapp in a remote server with no direct java component class level access? Thus, it's always the best practice to make a REST API call to a backend server from jaggery end to perform the same.